src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 0.3s ease;
```

Comment:
Specify what will be transitioned, you can use all, or deal with transition for color and transform separately